ios/RCTWebRTC/ScreenCapturer.m

```diff
@@ -186,20 +188,38 @@ - (void)readBytesFromStream:(NSInputStream *)stream {
     }
 }
 
-- (void)didCaptureVideoFrame:(CVPixelBufferRef)pixelBuffer {
+- (void)didCaptureVideoFrame:(Message *)message {
```

Comment:
I would add the orientation as a method param. It's more objc style this way.

Reply:
Pass the pixel buffer and orientation as 2 parameters? Sure, can do!